src/library/dlgtrackinfo.cpp

```diff
@@ -45,6 +45,8 @@ void DlgTrackInfo::init() {
 
     m_pTagFetcher.reset(new DlgTagFetcher(this, m_pTrackModel));
     if (m_pTrackModel) {
+        // The default parameter in QPushButton::clicked signal is false,
+        // so slotNext needs to be called explicitly to maintain true.
```

Comment:
This is not straightforward with the implicit default values for slotNext and slotPrev. Please remove the implicit default from the header file definitions of these functions and explicitly call `slotNext(true)`/`slotPrev(true)` in these lambdas.